### DISCUSSION TITLE
Footer with links to contact and data protection (maybe more links)

### DISCUSSION THREAD

**Discussion by saschafoerster** [lukevella/rallly#571] [Ideas]
I need to add a footer with (at least) two links, reachable from every page: 
"Impress/contact" and
"Data protection", 
so I can link to our company contact details and to our own data protection guidelines, as this is obligatory for online services, even though they are behind a login. And especially if we then invite customers or other people, we need this to be compliant with GDPR and other laws. 
In a basic setting I would be happy with an .env-setting, where I can add these links with title and link. 
In a later stage, I guess people would like to have an admin-backend, where they can change more settings, like colors, fonts and everything else needed for white labeling.

**Comment by aquatic7** [comment]
Did you figure out how to do this?

**Reply by saschafoerster** [reply]
Nope, not yet.
